src/moin/cli/_tests/test_modify_item.py

```diff
@@ -36,7 +36,6 @@ def test_welcome(welcome):
     assert_p_succcess(welcome)
 
 
-@pytest.mark.skip(reason="1471 fails on debian")
```

Comment:
I didn't see any logs from this failure, so enabled without modification, please provide logs from failures if this test continues to be troublesome